ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -67,8 +74,31 @@ function Get-sbRDSession {
         [string[]]$UserName = $null
     )
 
+    Begin {
+        if (-not($ConnectionBroker)) {
+            Write-Verbose "[BEGIN  :] No Connection Broker specified, querying and connecting to local host FQDN."
+            $computerinfo = Get-CimInstance -ClassName Win32_ComputerSystem
+            $ConnectionBroker = "$($computerInfo.DNSHostName).$($computerInfo.Domain)"
+        } else {
+        }
+
+        if (-not($SkipCheck)) {
```

Comment:
Avoid using `-not` if you can, as it's easier for people to follow flow.
Consider using:
```
if ($SkipCheck.IsPresent) { }
```

Reply:
Understood - will adjust. On the `connectionbroker` example above - would you agree that's a more appropriate use of it?

Reply:
yeah that's fine